package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "file-box",
-  "version": "1.4.13",
+  "version": "1.4.14",
```

Comment:
Bump the minor version instead of the patch, to make this change/update a new minor version, please.

Reply:
It's better to comment on others' questions and let others confirm whether the problem has been solved or not, instead of resolving them without any feedback.

Please bump the minor version.